INMOOV-Head-Control/INMOOV-Head-Control.ino

```diff
@@ -1,7 +1,67 @@
-void setup(){
+#include "StandardServo.h" //  Class
+#include "Head.h"          //  Class
 
+// class, variable object name, pin #, minPWMTime, maxPWMTime, degreeRange, outputGearRatio, reverseDirection, minRotation, maxRotation, absolute starting position
+StandardServo servo_1(3, 500, 2500, 180, 1, 0, 0, 180, 90);
+StandardServo servo_2(5, 500, 2500, 180, 1, 0, 0, 180, 90);
+StandardServo servo_3(6, 500, 2500, 180, 1, 0, 0, 180, 90);
+StandardServo servo_4(7, 500, 2500, 180, 1, 0, 0, 180, 90);
+StandardServo servo_5(8, 500, 2500, 180, 1, 0, 0, 180, 90);
+StandardServo servo_6(9, 500, 2500, 180, 1, 0, 0, 180, 90);
+
+// class, variable object name,
+Head head(servo_1, // objects included in class
+          servo_2, //        "
+          servo_3, //        "
+          servo_4, //        "
+          servo_5, //        "
+          servo_6, //        "
+          40);     //        update frequency
+
+int duration = 500;           //  Time for how long the movement will take place
+volatile bool interrupt1 = 0; //  Variable for the state of the interrupt
```

Comment:
Update the variable name to be more descriptive, for example if we are using a hardware interrupt on a button, call the interrupt as "buttonInterruptFlag".

Reply:
This name was chossen to be generic example placeholder to show how a interrupt is implemented. This intterrupt could be triggered by various hardware or software events and would be named more specifically in final code.